buildAndRunSite.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-jekyll serve --incremental --watch --config _config.yml,_config_en.yml 
\ No newline at end of file
+jekyll serve --incremental --watch --config _config_default.yml,_config_en.yml
```

Comment:
will this end up generating the site for both `en` and `es`, and make `en` default - I thought we had agreed upon using ` _config_default.yml,_config_en.yml`, which does generate both `en` and `es`?